etcd/etcd-live-http-to-https-migration.md

```diff
@@ -1,73 +1,73 @@
-# etcd Live HTTP to HTTPS migration
+# Enabling HTTPS in an Existing etcd Cluster
 
-If you already have existing etcd cluster which uses insecure HTTP connections and you would like to switch from HTTP to HTTPS and in addition enable TLS certificate authentication, you have to follow instruction below.
+This guide outlines the process of migrating an existing etcd cluster from HTTP communication to encrypted HTTPS. For added security, it also shows how to require TLS client-server certificates to authenticate connections.
 
 ## Prepare Cluster Components
 
-### Check insecure port availability
+### Check Insecure Port Availability
 
-By default fleet and flannel use two etcd client ports: 2379 (IANA port assignment) and 4001 (legacy port). Last one is used for etcd v0.4 backward compatibility. In this example we will use 4001 TCP port for insecure HTTP connections and our etcd members will listen that port only on local interface for security reason. This will allow us to switch transparently to HTTPS configuration without services downtime and simplify configuration
+By default, etcd communicates with clients over two ports: 2379, the current and official IANA port designation, and 4001, for clients who may implement versions of the protocol older than 0.4. We leverage this quirk of legacy support to migrate a running cluster from insecure plain-HTTP communication (on the old port, 4001) to encrypted HTTPS (on the current port, 2379) without cluster downtime. We restrict the legacy communication on port 4001 to the local interface.
 
-If you've configured flannel, fleetd or other local components to use custom or 2379 port only, you have to configure them to use 4001 TCP port.
+If you've configured flannel, fleet, or other components to use custom ports, or 2379 only, they will be reconfigured to use port 4001.
```

Comment:
They can. But you have to reconfigure them and this doc will be much longer. In addition we have to restart fleet with a hook `kill -9 $(pidof fleetd); systemctl restart fleet` to keep existing scheduled units. And I would like to avoid that.